packages/components/src/components/timeline/timeline.jsx

```diff
@@ -12,14 +12,15 @@ const Oval = ({ children }) => {
     );
 };
 
-const Timeline = ({ children, ...props }) => {
+const Timeline = ({ children, inactive_items, ...props }) => {
     return (
```

Comment:
The feature for disabling items is added to timeline. The array with number of options for disabling is supposed to be passed to the main <TimeLine> component